src/Symfony/Component/Ldap/Entry.php

```diff
@@ -22,9 +22,24 @@ class Entry
     public function __construct(string $dn, array $attributes = array())
     {
         $this->dn = $dn;
+        $attributes = $this->resetValueIndex($attributes);
         $this->attributes = $attributes;
     }
 
+    /**
+     * Resets the index for array value
+     *
+     * @param array $attributes
```

Comment:
You can remove this by typehinting the attributes.

Reply:
As far as I see in the Entry class, this is used for all other methods, why should I omit this. typehinting will be added.